src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -141,6 +177,18 @@
               {
                 "value": "8885552222"
               }
+            ],
+            "relationship": [
+              {
+                "coding": [
+                  {
+                    "code": "Y",
+                    "display": "Y",
+                    "system": "Y",
+                    "version": "Y"
```

Comment:
Can we just fill in `code` and `display` for coding? It looks that `system` is expecting an uri and `version` is not expecting a code as well.

Reply:
I agree, so we can just keep the `system` and `version` empty?

Reply:
I think `code` and `display` would be enough.

Reply:
Sure, it's already been updated.